diff --git a/survey/default.nix b/survey/default.nix
index b28606c..c9c51cc 100644
--- a/survey/default.nix
+++ b/survey/default.nix
@@ -1053,6 +1053,13 @@ let
                   [ final.openssl ]
                   "--libs openssl";
 
+                  hledger = 
+                  addStaticLinkerFlagsWithPkgconfig
+                  super.hledger
+                  [ final.hledger-lib ]
+                  "--libs hledger-lib"
+                  ;
+
               xml-to-json =
                 addStaticLinkerFlagsWithPkgconfig
                   super.xml-to-json
